### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by dzianis-l**
All the steps done's been listed in the README file

Task 3.1 - done fully.
Task 3.2 - done fully.
Task 3.3 - done fully.

All optional tasks are accomplished.

- [Cloudfront FE side](https://d2d25cyeqtftsg.cloudfront.net/)
- [/products](https://xun888d5wf.execute-api.eu-west-1.amazonaws.com/dev/products)
- [/products/{id}](https://xun888d5wf.execute-api.eu-west-1.amazonaws.com/dev/products/m6Gkc19AVw)
- [swagger](https://eqkv6cw9rf.execute-api.eu-west-1.amazonaws.com/swagger.json)
- [swagger-ui](https://eqkv6cw9rf.execute-api.eu-west-1.amazonaws.com/swagger)

### CODE REVIEW

File: services/product-service/src/functions/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default as hello } from "./getProductsList"
```
Comment (AL1088717) [inline services/product-service/src/functions/index.ts:1]: You should probably give a proper name instead of default 'hello'

**Review by AL1088717 — COMMENTED** [review]
@dzianis-l 

Base tasks:
- (!) currently FE is not fully integrated with BE, getting cors error on product/{productId} request (see FE PR comment)

Additional tasks:
- async avait OK +5
- es6 modules OK +5
- esbuild OK +4
- (!) swagger almost OK 
- unit tests OK +4
- separate files for lambdas OK +4
- main error scenarios OK +4

Please provide a proper url for your swagger-ui, instead of raw json (eg. [swagger ui](https://eqkv6cw9rf.execute-api.eu-west-1.amazonaws.com/swagger)). In the ui you should be able to test your requests and receive correct responses (now requests fail).

Good job, very nice drumstick store! Please fix the above comments and everything is perfect :)

File: services/product-service/src/functions/index.ts (hunk 1 above)
Comment (dzianis-l) [reply services/product-service/src/functions/index.ts:1]: Thanks for the notice!

Fixed

**Comment by dzianis-l** [comment]
> @dzianis-l
> 
> Base tasks:
> 
> * (!) currently FE is not fully integrated with BE, getting cors error on product/{productId} request (see FE PR comment)
> 
> Additional tasks:
> 
> * async avait OK +5
> * es6 modules OK +5
> * esbuild OK +4
> * (!) swagger almost OK
> * unit tests OK +4
> * separate files for lambdas OK +4
> * main error scenarios OK +4
> 
> Please provide a proper url for your swagger-ui, instead of raw json (eg. [swagger ui](https://eqkv6cw9rf.execute-api.eu-west-1.amazonaws.com/swagger)). In the ui you should be able to test your requests and receive correct responses (now requests fail).
> 
> Good job, very nice drumstick store! Please fix the above comments and everything is perfect :)

Many thanks for the review!

On the QA session for the task 3 it has been stated to not overcomplicate things.
Regarding the swagger support it's been said by the experts that even plain .json swagger file is enough, leave alone its integration.

I've provided the UI link in both README and here in the PR description though.

Thanks!

**Pull request merged**